package.json

```diff
@@ -43,5 +43,6 @@
     "patchedDependencies": {
       "oclif@4.16.0": "patches/oclif@4.16.0.patch"
     }
-  }
+  },
+  "packageManager": "pnpm@9.15.0+sha512.76e2379760a4328ec4415815bcd6628dee727af3779aaa4c914e3944156c4299921a89f976381ee107d41f12cfa4b66681ca9c718f0668fa0831ed4c6d8ba56c"
```

Comment:
Could you remove this `packageManager` field? Seems to be breaking github actions.

Reply:
removed